src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -479,23 +479,20 @@ def post_process_panoptic_segmentation(
                 The object mask threshold.
             overlap_mask_area_threshold (`float`, *optional*, defaults to 0.8):
                 The overlap mask area threshold to use.
-            is_thing_map (`Dict[int, bool]`, *optional*):
-                Dictionary mapping class indices to either `True` or `False`, depending on whether or not they are a
-                thing. If not set, defaults to the `is_thing_map` of COCO panoptic.
+            label_ids_to_fuse (`Set[int]`, *optional*):
+                A set of integers, if a label id is present, all its instances will be fused together.
```

Comment:
```suggestion
                The labels in this state will have all their instances be fused together. For instance we could say there can only be one sky in an image, but several persons, so the label ID for sky would be in that set, but not the one for person.
```
A set of integers -> that part is said in the type. I expanded since I found it a bit unclear.